guides/hosting/installation-updates/high-load-setup.md

```diff
@@ -0,0 +1,76 @@
+# Enterprise project setup
```

Comment:
The page title is still enterprise project setup. Will this work good together with the search?

Reply:
@Christian-Rades